observations/README.md

```diff
@@ -0,0 +1,1279 @@
+| availabilityTime \[@value]                                               | same as the Parent Compound Statement's `availatilityTime`                                                                                                                                                                                             |
+| value                                                                    | `Observation.component[index].valueQuantity`                                                                                                                                                                                                           |
+| referenceRange / referenceInterpretationRange / text                     | `Observation.component[index].referenceRange.text`                                                                                                                                                                                                     |
+| referenceRange / referenceInterpretationRang / value                     | `Observation.component[index].referenceRange.value`                                                                                                                                                                                                    |
+
+### Narrative Statement components
+
+A blood pressure may have a `NarrativeStatement` component if the parent observation has a comment or if `Observation.component.dataAbsentReason`, 
+`Observation.bodySite` or `Observation.component.interpretation` are populated.   
+
+| Mapped to (XML HL7 CompoundStatement / component / NarrativeStatement) | Mapped from (JSON FHIR / other source )                                                                                                              |
+|------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| id                                                                     | Unique ID generated by the adaptor                                                                                                                   |
+| text                                                                   | Concatenated from `Observation.comment`, `Observation.bodySite`, `Observation.component.dataAbsentReason` and `Observation.component.interpretation` |
+| statusCode                                                             | fixed value = `"COMPLETE"`                                                                                                                           |
+| availabilityTime \[@value]                                             | same as the Parent Compound Statement's `availatilityTime`                                                                                           |
```

Comment:
Typo in `availatilityTime`

Reply:
fixed, thanks

Reply:
I've noticed the same typo in a couple of other places 😬